library/registry

```diff
@@ -1,8 +1,11 @@
 Maintainers: Milos Gajdos (@milosgajdos),
              Sebastiaan van Stijn (@thajeztah)
 GitRepo: https://github.com/docker/distribution-library-image.git
+
+Tags: 3.0.0-alpha.1
+Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
 GitFetch: refs/heads/master
-GitCommit: 39dd72feaab7066334829d6945c54bc51a0aee98
+GitCommit: 6356e8dbe1969f7340ca25a9874faa26e6e6f71d
 
 Tags: 2.8.3, 2.8, 2, latest
 Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
```

Comment:
And then, since you want to still include 2.x (for now), it'll still need an appropriate `GitCommit:` value (that points at the 2.x `Dockerfile`): :heart:

```suggestion
Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
GitCommit: 39dd72feaab7066334829d6945c54bc51a0aee98
```